src/tests/unit/dhcpv6_rfc3319.txt

```diff
@@ -0,0 +1,36 @@
+#
+#  Test vectors for DHCPv6 attributes for (SIP) Servers
+#
+load dhcpv6
+dictionary-load dhcpv6
+
+#
+#  3.1  SIP Servers Domain Name List
+#
+
+#  domain = mysip.com
+encode-pair SIP-Server-Domain-Name-List = mysip.com
+data 00 15 00 09 6d 79 73 69 70 2e 63 6f 6d
```

Comment:
RFC 3319 Section 3.1 says:

The option length is followed by a sequence of labels, encoded
   according to Section 3.1 of RFC 1035 [5], quoted below:

      "Domain names in messages are expressed in terms of a sequence of
      labels.  Each label is represented as a one octet length field
      followed by that number of octets.  Since every domain name ends
 with the null label of the root, a domain name is terminated by a
      length byte of zero.  The high order two bits of every length
      octet must be zero, and the remaining six bits of the length field
      limit the label to 63 octets or less.  To simplify
      implementations, the total length of a domain name (i.e., label
      octets and label length octets) is restricted to 255 octets or
      less."

      RFC 1035 encoding was chosen to accommodate future
      internationalized domain name mechanisms.

i.e. this is *not* data type `string` or `octets`.  It's data type DNS resource record.  The encoding here is *not* just the bare string, but the DNS resource record encoding.

This test should be a failed test, because we don't currently support encoding DNS resource records.